### PR TITLE
[#94] Add note to README about 4.3.2+ servers (main)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # iRODS GenQuery2 Parser
 
+**IMPORTANT: iRODS 4.3.2 has built-in support for GenQuery2. DO NOT install binaries produced by this project on 4.3.2+ servers.**
```
Comment (alanking) [inline README.md:3]:
```suggestion
**IMPORTANT: iRODS 4.3.2 has built-in support for GenQuery2. DO NOT install packages or binaries produced by this project on 4.3.2+ servers.**
```

**Review by alanking — APPROVED** [review]
One minor clarification. Take or leave it

File: README.md (hunk 1 above)
Comment (korydraughn) [reply README.md]: Done.

**Review by alanking — APPROVED** [review]
Yeap. # it.

**Comment by korydraughn** [comment]
#'d

**Pull request merged**